doc/user_doc/vic_admin/upgrade_ui_plugin.md

```diff
@@ -1,19 +1,19 @@
-# Upgrade the vSphere Integrated Containers Engine Plug-In #
+# Upgrade the Flex-Based vSphere Web Client Plug-In #
```

Comment:
This is just a note, but we don't bring additional features to the Flex plugin, so we probably want to either have a message regarding that, or drop this section. What's your thought?